For local try user `python try.py`; 
When you are using try you have to mentioned that in sys_conf > running_mode
i.e. 
running_mode = 'TRY'


For running bot in production mode sys>conf > running_mode = 'PRO'
and you have to use `python DialerRun.py`

